chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/urlStatus/UrlStatusDataCollector.java

```diff
@@ -285,24 +271,15 @@ private void completeSetup() {
             LOGGER.error("The spring environment was null.");
         }
 
-        TrustStrategy acceptingTrustStrategy = (X509Certificate[] chain, String authType) -> true;
-        SSLContext sslContext = null;
-        try {
-            sslContext = org.apache.http.ssl.SSLContexts.custom().loadTrustMaterial(null, acceptingTrustStrategy).build();
-        } catch (Exception ex) {
-            LOGGER.error("Could not create ssl context; https requests may fail.", ex);
-        }
-
-        SSLConnectionSocketFactory csf = new SSLConnectionSocketFactory(sslContext != null ? sslContext : SSLContexts.createDefault());
-        httpClient = HttpClients.custom()
-                .setSSLSocketFactory(csf)
-                .setDefaultRequestConfig(RequestConfig.custom()
-                        .setConnectionRequestTimeout(requestTimeoutSeconds * SECONDS_TO_MILLIS)
-                        .setConnectTimeout(connectTimeoutSeconds * SECONDS_TO_MILLIS)
-                        .setSocketTimeout(connectTimeoutSeconds * SECONDS_TO_MILLIS)
-                        .build())
-                .build();
-
+        webClient = new WebClient(BrowserVersion.CHROME, false, null, -1);
+        webClient.getOptions().setRedirectEnabled(true);
```

Comment:
I had thought we weren't supporting redirects as "valid" URLs. Or is that only for the Service Base URL List check?